docs/Overview.md

```diff
@@ -75,4 +75,10 @@ Because Draft.js supports unicode, you must have the following meta tag in the `
 <meta charset="utf-8" />
 ```
 
+`Draft.css` should be included when rendering the editor. Learn more about [why](/docs/advanced-topics-issues-and-pitfalls.html#missing-draft-css).
+
+```js
```

Comment:
Yeah, I think your suggestion makes sense. What I'm doing here could be giving the impression that this is the standard way to include the css file, which, as you pointed out, is not true. I will just remove the code example.